flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/internal/ShowCreateUtil.java

```diff
@@ -82,7 +82,11 @@ static String getColumnString(Column column) {
                                 sb.append(e);
                             });
         }
-        // TODO: Print the column comment until FLINK-18958 is fixed
+        Optional<String> comment = column.getComment();
+        if (comment.isPresent()) {
+            sb.append(" ");
+            sb.append(String.format("comment '%s'", comment.get()));
```

Comment:
Please use the upper-case of "comment" and the comment string should be escaped for `'` character (please add tests for it).

Reply:
Thanks for your detailed review, I have fixed it and rebased to master branch.